app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,39 @@
+class InvitationsController < ApplicationController
+	def new_invitation	
+		@id = User.where(phone_number: params[:invited_phone_number]).first.id
+		@invitation = Invitation.new(invitation_params.merge(user_id: @id,user_sent_by_id: params[:user_id], group_id: params[:group_id], accepted: false, rejected: false))
```

Comment:
Maybe you should replace `accepted` and `rejected` with `status`. And have a value of accepted or rejected.